dmsite/venv/src/SearchResultsPage/SearchResultsCell.jsx

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
-import { ListGroupItem } from 'reactstrap';
+import {
+  ListGroupItem,
+  ListGroupItemHeading,
+  ListGroupItemText,
+} from 'reactstrap';
```

Comment:
Are we going to switch over to antd instead of reactstrap?

Reply:
yes